.github/workflows/update-ref-tests.yml

```diff
@@ -63,30 +73,52 @@ jobs:
             
             - [ ] All tests must pass
 
-      - name: Create Asana task when workflow succeeded
-        if: ${{ steps.update-check.outcome == 'failure' }}
-        uses: honeycombio/gha-create-asana-task@main
+      - name: Create Asana task in Android App project
+        if: ${{ steps.update-check.outputs.outcome == 'failure' }}
+        id: create-task
+        uses: malmstein/github-asana-action@master
         with:
-          asana-secret: ${{ secrets.GH_ASANA_SECRET }}
-          asana-workspace-id: ${{ secrets.GH_ASANA_WORKSPACE_ID }}
-          asana-project-id: ${{ secrets.GH_ASANA_AOR_PROJECT_ID }}
-          asana-section-id: ${{ secrets.GH_ASANA_INCOMING_ID }}
-          asana-task-name: Update reference tests to latest version
+          asana-pat: ${{ secrets.GH_ASANA_SECRET }}
+          asana-project: ${{ vars.GH_ANDROID_APP_PROJECT_ID }}
+          asana-section: ${{ vars.GH_ANDROID_APP_INCOMING_SECTION_ID }}
+          asana-task-name: Update reference tests to version ${{ steps.latest-release.outputs.version }}
           asana-task-description: |
             Reference tests have been updated and a PR created.
             
             If tests failed check out https://app.asana.com/0/0/1203766026095653/f for further information on what to do next.
             
             See ${{ steps.create-pr.outputs.pull-request-url }}
+          action: 'create-asana-task'
+
+      - name: Add Asana task to Release Board project
+        if: ${{ steps.update-check.outputs.outcome == 'failure' }}
+        uses: malmstein/github-asana-action@master
+        with:
+          asana-pat: ${{ secrets.GH_ASANA_SECRET }}
+          asana-project: ${{ vars.GH_ANDROID_RELEASE_BOARD_PROJECT_ID }}
+          asana-section: ${{ vars.GH_ANDROID_RELEASE_BOARD_PR_SECTION_ID }}
+          asana-task-id: ${{ steps.create-task.outputs.taskId }}
+          action: 'add-task-asana-project'
+
+      - name: Update PR description with Asana task
```

Comment:
This step failed with

```
calling add-task-pr-description
/home/runner/work/_actions/malmstein/github-asana-action/master/node_modules/@octokit/request/dist-node/index.js:122
      const error = new import_request_error.RequestError(toErrorMessage(data), status, {
```

See error in https://github.com/duckduckgo/Android/actions/runs/5452587649/jobs/9920294125

Reply:
Same, there’s no PR to update, that’s why the step fails.

Reply:
when you are right, you are right